src/components/counter/Counter.tsx

```diff
@@ -1,22 +1,36 @@
-import React, { useState } from 'react';
-import styled from 'styled-components';
-
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { isAMultipleOfFive } from '../../utils/numbers'
 
 const Counter = () => {
-  const [currentValue, setCurrentValue] = useState(0)
-  const incrementValue = () => setCurrentValue(currentValue + 1);
+  const [currentValue, setCurrentValue] = useState<number>(0)
+  const [showMessage, setShowMessage] = useState<boolean>(false)
+  const incrementValue = () => setCurrentValue(currentValue + 1)
+  useEffect(() => {
+    if (isAMultipleOfFive(currentValue)) return setShowMessage(true)
+    setShowMessage(false)
+  }, [currentValue])
   // Implement a useEffect hook that, when the count hits a multiple of 5 it
   // renders a message
```

Comment:
Probably a good idea to remove commens that re-state what the code is already doing.